app/assets/stylesheets/modules/_document_access_badge.scss

```diff
@@ -43,4 +61,39 @@
   img {
     height: 1.4rem
   }
+
+}
+
+.banner-content {
+  flex-grow: 1;
+  background-color: #edf6ee;
+  overflow: hidden;
+  color: #009900;
+  font-weight: bold;
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 0.75em;
+  padding: 10px 10px 10px 30px;
+
+  span {
+    padding-right: 20px;
+  }
+}
+
+.slant {
+  // width: 62px;
+  // // background-image: image-url("slant.png");
```

Comment:
question: do we still need the slant.png file since it's not being used in the css? Also, Is the online-access-bg image being used?